extensions/wikia/JWPlayerTag/scripts/jwplayertag.js

```diff
@@ -0,0 +1,45 @@
+require(['jquery'], function ($) {
+	var parserTagSelector = '.jwplayer-in-article-video .jwplayer-container',
+		jwVideoDataUrl = 'https://cdn.jwplayer.com/v2/media/';
+
+	function init() {
+		var parserTags = document.querySelectorAll(parserTagSelector);
+debugger;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this debugger statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ADebuggerStatement)

[project-id]: wikia:app